src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -644,45 +607,25 @@ const promise = makeAPICall();
 API.cancel(promise, "my error message");
 ```
 
-### Recap
-You have implemented a few mutations to change your data. 
-
-[comment]: # (ADD TO RECAP a tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
-[comment]: # (RENE, more detail needed here? We're not really addressing the tie-in or real-world use case or the benefit of what we've done in this module.)
+**Recap:** You have implemented a few mutations to change your data. 
 
 ## Read application data
 
-In this job, you will read application data using the Amplify GraphQL client. This will include understanding the differences between listing and getting data, how to filter this data, and how to paginate your data. We will also show you how to cancel these requests when needed.
-
-[comment]: # (RENE, suggest we revise this to something like: In this job, you will learn to read application data using Amplify's GraphQL client. Specifically, you will learn:
-
-> - The difference between listing data and getting a single item
-> - How to filter query results to get just the data you need
-> - How to paginate results so you don't retrieve too much data
-> - How to cancel requests when no longer needed)
-
-> **Before you begin, you will need:**
-> - Your application connected to the API
-> - Populated data to query
+You can read application data using the Amplify GraphQL client. In this next section we will review the difference between listing data and getting data, how to filter query results to get just the data you need, and how to paginate results to make you're data more manageable. We will also show you how to cancel these requests when needed. Before you begin you will need your application connected to the API and some populated data to query.
```

Comment:
Again we are changing the template to have the starting prereqs at the top and then reference the other sections. I did not link this one and the previous since these will be modularized. But users can find this information on the same page. I also removed the proposed bullet points to keep the narrative intro structure.

Reply:
LGTM.